appveyor.yml

```diff
@@ -45,16 +46,12 @@ install:
 - cmd: cinst wget
 - ps: Import-Module .\appveyor.psm1
 - ps: SetupPhpVersionString
+- ps: AppendSessionPath
 
 build_script:
-- set VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\
-- if "%VC_VER%" == "vc14" set VSCOMNTOOLS=%VS120COMNTOOLS%
-- if "%VC_VER%" == "vc15" set VSCOMNTOOLS=%VS140COMNTOOLS%
-- set PATH=%PATH%;C:\projects\php-sdk\bin;C:\projects\php\bin;C:\projects\php
-- set TEST_PHP_EXECUTABLE=C:\projects\php\bin\php.exe
-- if "%platform%" == "x86" call "%VSCOMNTOOLS%\..\..\VC\vcvarsall.bat"
-- if "%platform%" == "x64" "C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd" /x64
-- if "%platform%" == "x64" call "%VSCOMNTOOLS%\..\..\VC\vcvarsall.bat" x86_amd64
+- ps: InitializeBuildVars
+- '"%VSCOMNTOOLS%\VsDevCmd" %PLATFORM%'
```

Comment:
Does this exist for vc14 and vc12? The CI passed so I suppose it must, but docs elsewhere imply that it doesn't.

Reply:
@jbboehr Actually I set `%VSCOMNTOOLS%` *before*. So yes `VsDevCmd` exist for vc14 and vc12